src/test/java/com/conveyal/r5/point_to_point/PointToPointRouterServerTest.java

```diff
@@ -0,0 +1,429 @@
+package com.conveyal.r5.point_to_point;
+
+import com.google.common.hash.Hashing;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.ReadContext;
+import net.minidev.json.JSONArray;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+
+import static com.google.common.io.Files.asByteSource;
+import static io.restassured.RestAssured.given;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.number.OrderingComparison.greaterThan;
+import static org.hamcrest.number.OrderingComparison.greaterThanOrEqualTo;
+
+public class PointToPointRouterServerTest {
+    private static final String resourcesDir = "./src/test/resources/com/conveyal/r5/point_to_point";
+
+    /**
+     * Start up a point to point route server that will be queried in various tests
+     */
+    @BeforeClass
+    public static void setUp() throws IOException {
+        // create a temporary director and copy needed files there
+        String[] files = {"network.dat", "osm.mapdb", "osm.mapdb.p"};
```

Comment:
These three binary files are included in the test resources. I tend to see them as intermediate results whose format is subject to change, with the original source data being the GTFS and OSM data. I'd like to avoid checking intermediate results in (especially the MapDB) if possible.

Reply:
Fixed in https://github.com/conveyal/r5/pull/345/commits/908d331ec291f2802cd484407d542022f5cd304d